### DISCUSSION TITLE
where is the link for download all controls Demo/Gallery ?

### DISCUSSION THREAD

**Discussion by sgf** [AvaloniaUI/Avalonia#17556] [Q&A] [answered]
I would like to have a more intuitive understanding of the richness of all the control categories and functions of Avalonia.

**Comment by stevemonaco** [comment]
There's no binary distribution of the `ControlCatalog`. The source is in the Avalonia repo, so you need to clone Avalonia and build both. This is relatively easy.

Follow https://github.com/AvaloniaUI/Avalonia/blob/master/Documentation/build.md, ensure you've updated submodules, open the `Avalonia.Desktop.slnf` so you don't need mobile/wasm workloads, and build. Then set `ControlCatalog` as startup project.

**Comment by rabbitism** [comment, marked as answer]
Actually we do have a separated repo for ControlCatalog, but needs some updates. 
https://github.com/AvaloniaUI/ControlCatalogStandalone
@avaloniaui-team

**Reply by stevemonaco** [reply]
Ah, I didn't know about that. It's been discussed before about pulling ControlCatalog out of Avalonia itself, but that leads to unexpected breaks and more difficult maintenance than a monorepo. Besides inevitably becoming out-of-date.
